README.md

```diff
@@ -5,6 +5,8 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 ![Version](https://img.shields.io/maven-central/v/com.hotels.styx/styx-server.svg)
 
+<img width="460" align="right" alt="Styx Logo" src="./docs/assets/styx-logo.svg">
```

Comment:
Looking at the page, IMO the logo would suit better in the top of README, in place of the top-level STYX header (`# STYX`).